pallets/connectors/src/message.rs

```diff
@@ -290,8 +290,8 @@ where
 	/// Schedules an EVM address to become rely-able by the gateway. Intended to be used via governance to execute EVM spells.
 	///
 	/// Directionality: Centrifuge -> EVM Domain.
-	ScheduleRely {
-		usr: Address,
+	ScheduleUpgrade {
+		user: Address,
```

Comment:
maybe given that this not actually a `user` address afterall but a spell contract address, we could make name this either `spell` or `contract`. wdyt?

Reply:
Yah good point. Contract makes the most sense I think